pkg/dao/helpers.go

```diff
@@ -0,0 +1,40 @@
+package dao
+
+import "strings"
+
+func convertSortByToSQL(SortBy string, SortMap map[string]string) string {
```

Comment:
you could add a test just for this function

Reply:
Rog!